obs_status.yml

```diff
@@ -0,0 +1,2147 @@
+  mag_aca_err: 0.1
+- agasc_id: 417611952
+  mag_aca: 5.0  # need to check
+  mag_aca_err: 0.1
+- agasc_id: 444743456
+  mag_aca: 5.0
+  mag_aca_err: 0.1
+- agasc_id: 450103168
+  mag_aca: 12.2
+  mag_aca_err: 0.1
+- agasc_id: 497824592
+  mag_aca: 13.078
+  mag_aca_err: 0.1
+- agasc_id: 505545160
+  mag_aca: 10.8
+  mag_aca_err: 0.1
+- agasc_id: 612370792
+  mag_aca: 12.177  # need to check, not from starcheck report
+  mag_aca_err: 0.1
+- agasc_id: 614482704
+  mag_aca: 11.25
+  mag_aca_err: 0.1
+- agasc_id: 615257632
+  mag_aca: 11.031
+  mag_aca_err: 0.1
+- agasc_id: 615914552
+  mag_aca: 10.797
+  mag_aca_err: 0.1
+- agasc_id: 616699752
+  mag_aca: 11.484
+  mag_aca_err: 0.1
+- agasc_id: 626667256
+  mag_aca: 10.85
+  mag_aca_err: 0.1
+- agasc_id: 639635448
+  mag_aca: 11.2
+  mag_aca_err: 0.1
+- agasc_id: 643702120
+  mag_aca: 11.4
+  mag_aca_err: 0.1
+- agasc_id: 647762704
+  mag_aca: 11.453
+  mag_aca_err: 0.1
+- agasc_id: 649070072
+  mag_aca: 12.719
+  mag_aca_err: 0.1
+- agasc_id: 652480464
+  mag_aca: 10.075
+  mag_aca_err: 0.1
+- agasc_id: 653529952
+  mag_aca: 10.844
+  mag_aca_err: 0.1
+- agasc_id: 653919168
+  mag_aca: 13.1
+  mag_aca_err: 0.1
+- agasc_id: 654050016
+  mag_aca: 11.625
+  mag_aca_err: 0.1
+- agasc_id: 656409216
+  mag_aca: 11.0
+  mag_aca_err: 0.1
+- agasc_id: 664279632
+  mag_aca: 11.672
+  mag_aca_err: 0.1
+- agasc_id: 686557584
+  mag_aca: 9.609
+  mag_aca_err: 0.1
+- agasc_id: 686557784
+  mag_aca: 9.047
+  mag_aca_err: 0.1
+- agasc_id: 689441328
+  mag_aca: 12.0
+  mag_aca_err: 0.1
+- agasc_id: 690625640
+  mag_aca: 11.2
+  mag_aca_err: 0.1
+- agasc_id: 693243160
+  mag_aca: 4.5
+  mag_aca_err: 0.1
+- agasc_id: 694561568
+  mag_aca: 9.8
+  mag_aca_err: 0.1
+- agasc_id: 720504096
+  mag_aca: 11.344
+  mag_aca_err: 0.1
+- agasc_id: 723651200
+  mag_aca: 12.0
+  mag_aca_err: 0.1
+- agasc_id: 723657296
+  mag_aca: 11.547
+  mag_aca_err: 0.1
+- agasc_id: 740693864
+  mag_aca: 11.875
+  mag_aca_err: 0.1
+- agasc_id: 763893760
+  mag_aca: 11.6
+  mag_aca_err: 0.1
+- agasc_id: 764150080
+  mag_aca: 12.0
+  mag_aca_err: 0.1
+- agasc_id: 771756656
+  mag_aca: 11.641
+  mag_aca_err: 0.1
+- agasc_id: 797847184
+  mag_aca: 12.3
+  mag_aca_err: 0.2
+- agasc_id: 798760200
+  mag_aca: 11.2
+  mag_aca_err: 0.1
+- agasc_id: 805966472
+  mag_aca: 11.703
+  mag_aca_err: 0.1
+- agasc_id: 821318496
+  mag_aca: 10.719
+  mag_aca_err: 0.1
+- agasc_id: 836372744
+  mag_aca: 11.609
+  mag_aca_err: 0.1
+- agasc_id: 836381648
+  mag_aca: 11.203
+  mag_aca_err: 0.1
+- agasc_id: 879768704
+  mag_aca: 10.859
+  mag_aca_err: 0.1
+- agasc_id: 891159184
+  mag_aca: 11.203
+  mag_aca_err: 0.1
+- agasc_id: 897878136
+  mag_aca: 10.75
+  mag_aca_err: 0.1
+- agasc_id: 898125104
+  mag_aca: 11.203
+  mag_aca_err: 0.1
+- agasc_id: 899950912
+  mag_aca: 11.688
+  mag_aca_err: 0.1
+- agasc_id: 901664112
+  mag_aca: 10.406
+  mag_aca_err: 0.1
+- agasc_id: 904010616
+  mag_aca: 10.902  # need to check, not from starcheck report
```

Comment:
I note starcheck has maxmag 10.876 but not sure if this entry for 904010616 should stay anyway.  Fine with whatever here.

Reply:
What do you mean by "not sure if this entry for 904010616 should stay anyway."?

Reply:
There was a star-obs record for this star with status = 0 with a reported mag close to this mag.  So the value of the 'mags' entry was questionable.